modules/javafx.graphics/src/test/java/test/javafx/scene/layout/SnappingTest.java

```diff
@@ -119,46 +129,46 @@ private void testContainerSnappingImpl(ContainerInstruction<Region> containerIns
             assertEquals(snappedPaddingX, container.minWidth(-1), 0.00001, className);
             assertEquals(snappedPaddingY, container.minHeight(-1), 0.00001, className);
         } else {
-            assertEquals(widthHeight + snappedPaddingX, container.minWidth(-1), 0.00001, className);
-            assertEquals(widthHeight + snappedPaddingY, container.minHeight(-1), 0.00001, className);
+            assertEquals(widthHeight + snappedPaddingX, container.minWidth(-1), DELTA, className);
+            assertEquals(widthHeight + snappedPaddingY, container.minHeight(-1), DELTA, className);
         }
 
-        assertEquals(widthHeight + snappedPaddingX, container.prefWidth(-1), 0.00001, className);
-        assertEquals(widthHeight + snappedPaddingY, container.prefHeight(-1), 0.00001, className);
+        assertEquals(widthHeight + snappedPaddingX, container.prefWidth(-1), DELTA, className);
+        assertEquals(widthHeight + snappedPaddingY, container.prefHeight(-1), DELTA, className);
     }
 
-    static Stream<ContainerInstruction<?>> getContainerInstruction() {
+    static Stream<ContainerCreator<?>> getContainerCreators() {
         // TODO: Create issues and fix snapping for all commented out layout containers below.
         // Note that the working layout container do not necessary use the optimized snappedXXXInsets() methods,
         // but instead snap the insets (again). This can be optimized as well.
         return Stream.of(
-//                new ContainerInstruction<>(Pane::new),
-//                new ContainerInstruction<>(StackPane::new),
-//                new ContainerInstruction<>(BorderPane::new),
-                new ContainerInstruction<>(HBox::new),
-                new ContainerInstruction<>(VBox::new),
-                new ContainerInstruction<>(node -> {
-                GridPane gridPane = new GridPane();
+                new ContainerCreator<>(HBox::new),
+                new ContainerCreator<>(VBox::new),
+                new ContainerCreator<>(node -> {
+                    GridPane gridPane = new GridPane();
                     gridPane.getChildren().add(node);
                     return gridPane;
                 })
-//                new ContainerInstruction<>(node -> {
+//                new ContainerCreator<>(Pane::new),
+//                new ContainerCreator<>(StackPane::new),
+//                new ContainerCreator<>(BorderPane::new),
+//                new ContainerCreator<>(node -> {
```

Comment:
1. Yes. We may want to create something like an 'epic'. I see that in the past it was called an 'umbrella issue'
2. Yeah we should, but the goal of this PR is to test all layout container, specified in `javafx.scene.layout`.

Other nodes needs more special test cases, e.g. can't be solved in a generic test. `Control` nodes are also mostly fine since the `layoutChildren` method in the `SkinBase` will receive the already snapped insets (so it is harder to do the wrong thing).

Reply:
could you please create an umbrella ticket listing all the types that need to be added to the test and/or fixed?

here is an example of such ticket (with an umbrella unicode):
[JDK-8241364](https://bugs.openjdk.org/browse/JDK-8241364)

Reply:
created https://bugs.openjdk.org/browse/JDK-8296609

Reply:
Should this list include the class `Region` and `FlowPane` too? I see these classes use non-snapped insets.

Reply:
`FlowPane` is  included (the last line). We don't need `Region`, as `Pane` directly extends from it without further modification. Thats why I decided to include only `Pane`. :)